### DISCUSSION TITLE
RN 0.68适配

### DISCUSSION THREAD

**Discussion by TOPWN** [funnyzak/react-native-quick-start#83] [General]
大佬有准备适配0.68新架构版本吗？

**Comment by funnyzak** [comment]
后续看下这块。
你先看下这个 https://www.react-native.cn/docs/upgrading ，试着升级看看。

**Comment by funnyzak** [comment]
@TOPWN RN已经升级到最新稳定版0.69.4。欢迎更新。

**Comment by TOPWN** [comment]
> @TOPWN RN已经升级到最新稳定版0.69.4。欢迎更新。
项目中引用的三方库应该存在某些还没有适配新架构吧？我看你Android端并没有真正开启新架构的使用
https://github.com/funnyzak/react-native-v2ex/blob/94ae38b28142abfa6be2db57922a77b597a2d323/android/gradle.properties#L42

**Comment by funnyzak** [comment]
@TOPWN 😄忘记开了。

目前在未开启新架构的前提下，升级RN发现：react-native-easy-toast 没有适配新架构。所以用 patch-package 做了个patch。

目前你有开启，发现什么问题或者经验分享吗？ 如果有时间的话，可以参与一起优化下项目

**Reply by TOPWN** [reply]
- 许多三方库对新架构的适配目前确实还存在一些问题，你这个项目是纯RN，升级还是方便的，我是Android开发，最近在搞原生和RN的混合开发，升级到新架构的复杂度很大，还在研究中。
- 我也挺想参与一起优化下项目的，但是最近抽不开身，后面看时间吧😄

**Reply by funnyzak** [reply]
@TOPWN  🤝

**Comment by funnyzak** [comment]
@TOPWN 看了下目前 newArchEnabled， 还属于实验阶段：

* https://reactnative.dev/docs/new-architecture-intro
* https://github.com/reactwg/react-native-new-architecture/discussions/8
如果要使用新架构，还得根据官方文档设置 https://reactnative.dev/docs/new-architecture-library-android
